src/lib/js-split-provider.ts

```diff
@@ -53,16 +54,17 @@ export class OpenFeatureSplitProvider implements Provider {
       flagKey,
       this.transformContext(context)
     );
+    const flagName = details.value.toLowerCase();
```

Comment:
`flagName` sounds like the feature flag name, but here the variable refers to the treatment value, right? So `flagValue` or `flagTreatment` are more appropriate names.